test/tools/Modules/PSSysLog/PSSysLog.psd1

```diff
@@ -0,0 +1,10 @@
+@{
```

Comment:
Should we add copyright header?

Reply:
none of the manifest files in the repo have copyright headers; they all have Copyright=""